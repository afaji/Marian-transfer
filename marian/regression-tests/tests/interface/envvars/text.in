India and Japan prime ministers meet in Tokyo
India &apos;s new prime minister , Nar@@ endra Modi , is meeting his Japanese counterpart , Shin@@ zo Abe , in Tokyo to discuss economic and security ties , on his first major foreign visit since winning May &apos;s election .
Mr Modi is on a five-@@ day trip to Japan to strengthen economic ties with the third largest economy in the world .
high on the agenda are plans for greater nuclear co-operation .
India is also reportedly hoping for a deal on defence collaboration between the two nations .
Kar@@ ra@@ tha police arrest 20-@@ year-old after high speed motorcycle chase
a motorcycle has been seized after it was ridden at 12@@ 5km @/@ h in a 70@@ km @/@ h zone and through bush@@ land to escape police in the Pil@@ bara .
traffic police on patrol in Kar@@ ra@@ tha this morning tried to pull over a blue motorcycle when they spotted it reaching 12@@ 5km @/@ h as it pulled out of a service station on Bath@@ gate Road .
police say the rider then failed to stop and continued on to Bur@@ ges@@ s Road before turning into bush@@ land , causing the officers to lose sight of it .
the motorcycle and a person matching the description of the rider was then spotted at a house on Wal@@ cott Way in Bulgar@@ ra .
