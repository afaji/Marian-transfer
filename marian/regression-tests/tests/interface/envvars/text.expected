Indien und Japan Ministerpräsidenten treffen sich in Tokio
Indiens neuer Premierminister Nar@@ endra Modi trifft sein jap@@ anisches Pendant , Shin@@ zo Abe , in Tokio , um über wirtschaftliche und Sicherheits@@ beziehungen zu diskutieren , auf seinem ersten wichtigen Auslands@@ besuch seit der Wahl im Mai .
Herr Modi ist auf einer fünft@@ ä@@ gigen Reise nach Japan , um die wirtschaftlichen Beziehungen zur dritt@@ größten Volkswirtschaft der Welt zu stärken .
auf der Tagesordnung stehen die Pläne für eine verstärkte nukleare Zusammenarbeit .
auch Indien hofft angeblich auf eine Einigung zur Verteidigungs@@ kooperation beider Nationen .
Kar@@ ra@@ tha Polizei nimmt 20-@@ Jährige nach Hochgeschwindigkeits@@ motor@@ en@@ jagd fest
ein Motorrad ist beschlagnahmt worden , nachdem es 12@@ 5km / h in einer 70@@ km / h-@@ Zone und durch Busch@@ land zur Flucht der Polizei in der Pil@@ bara ger@@ ut@@ scht war .
die Verkehrs@@ polizei auf Pat@@ rou@@ ille in Kar@@ ra@@ tha an diesem Vormittag versuchte , über ein blau@@ es Motorrad zu ziehen , als sie sie entdeckten , sie erreichte 12@@ ,5 / h , wie sie aus einer Tankstelle an der Bath@@ tor@@ straße aus@@ zog .
die Polizei meldet den Reiter dann nicht an und fuhr weiter bis zur Bur@@ ges@@ e-@@ Straße , bevor er in Busch@@ land fuhr , wodurch die Beamten aus den Augen verloren haben .
das Motorrad und ein Mensch Treffer der Beschreibung des Rei@@ ters wurde dann in einem Haus auf Wal@@ cott Way in Bulgar@@ ra ges@@ ichtet .
